.changeset/dull-oranges-tap.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': minor
+---
+
+add ability to configure filters when using EntityListDocsGrid
```

Comment:
Can you include an example of how to use this? Would be really helpful to have this here for those upgrading. Also, what would be even better is to add this to the docs themselves, perhaps here: https://backstage.io/docs/features/techdocs/how-to-guides

Reply:
Yeah, that's a good idea

I'll also improve the documentation around `TechDocsCustomHome` that techdocs provide but isn't documented 🤔

Reply:
I've updated the documentation, let me know if there is any further adjustments I should make 👀